src/AppBundle/Repository/EventWikiRepository.php

```diff
@@ -257,6 +258,56 @@ public function getPageIds(
         return $result ? array_map('intval', $result) : $result;
     }
 
+    /**
+     * Get the total pageviews count for a set of pages, from a given date until today. Optionally reduce to an average
+     * of the last 30 days.
+     * @param string $dbName
+     * @param EventWiki $wiki
+     * @param DateTime $start
+     * @param int[] $pageIds
+     * @param bool $getDailyAverage
+     * @return int
+     */
+    public function getPageviews(
+        string $dbName,
+        EventWiki $wiki,
+        DateTime $start,
+        array $pageIds,
+        bool $getDailyAverage = false
+    ): int {
+        if (0 === count($pageIds)) {
+            return 0;
+        }
+        $pageviewsRepo = new PageviewsRepository();
+        $recentDayCount = Event::AVAILABLE_METRICS['pages-improved-pageviews'];
+        $thirtyDaysAgo = (new DateTime())->sub(new DateInterval('P'.$recentDayCount.'D'));
```

Comment:
Since `$recentDayCount` isn't necessarily 30 days (could be changed later), I'd use a different variable name. Maybe `$offsetDate`?

Reply:
Good point! Fixed.